staging/src/k8s.io/kubectl/pkg/scale/scale.go

```diff
@@ -112,6 +113,16 @@ type genericScaler struct {
 	scaleNamespacer scaleclient.ScalesGetter
 }
 
+// objectForReplicas is used for encoding scale patch
+type objectForReplicas = struct {
```

Comment:
Move the type declaration for these into ScaleSimple.

Reply:
done